test/TestProjects/MgmtMultipleParentResource/Generated/AnotherParent.cs

```diff
@@ -416,11 +416,11 @@ public virtual AnotherParentUpdateOperation Update(AnotherParentUpdate anotherBo
             }
         }
 
-        /// <summary> Gets a list of AnotherParentChildBodies in the AnotherParent. </summary>
-        /// <returns> An object representing collection of AnotherParentChildBodies and their operations over a AnotherParent. </returns>
-        public AnotherParentChildBodyCollection GetAnotherParentChildBodies()
+        /// <summary> Gets a list of ChildBodyAnotherParents in the AnotherParent. </summary>
+        /// <returns> An object representing collection of ChildBodyAnotherParents and their operations over a AnotherParent. </returns>
+        public ChildBodyAnotherParentCollection GetChildBodyAnotherParents()
         {
-            return new AnotherParentChildBodyCollection(this);
+            return new ChildBodyAnotherParentCollection(this);
```

Comment:
why did renames happen in this project as well?

Reply:
Fixed